src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printList(Task[] list, int counter){
+        for(int i = 0; i < counter; i++){
+            System.out.print(i + 1 + ".");
+            printTask(list[i]);
+        }
+    }
+    public static void printTask(Task t){
+       System.out.println("[" + t.getStatusIcon()+ "] " + t.taskName);
+    }
+
+    public static int getTaskNumberArgument(String input){
```

Comment:
I like the descriptive method name. However, I think it's not entirely clear what a "task number argument" is supposed to be, so I'd suggest writing a comment above the method to help explain it.